src/assets/ar-busstops/scripts/places.js

```diff
@@ -109,3 +118,110 @@ function distanceBetweenCoords(lat1, lon1, lat2, lon2) {
 function degreesToRadians(degrees) {
     return degrees * Math.PI / 180;
 }
+
+// update gps-entity-place component to include more functionality
+// original: https://github.com/jeromeetienne/AR.js/blob/master/aframe/src/location-based/gps-entity-place.js
+
+AFRAME.registerComponent('gps-entity-place-busstop', {
```

Comment:
This particular part of the code seems to be already available since you have included `AR.js`library as script tag. What is purpose of duplicating it? instead can we reuse it by declaration as variable of some sort, is that possible.

Reply:
I will add an event listener later on to prevent code duplication.